README.md

```diff
@@ -32,39 +32,49 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the DOM?
 
+Document Object Model (DOM) allows you to manipulate html and css through javascript.
+
 2. What is an event?
 
+The JavaScript interacation with HTML is handled through events that occur when the user or the browser manipulates a page. When the page loads, it is called an event.
+
 3. What is an event listener?
 
+An event listener is a function that waits for an event to occur.
```

Comment:
And then returns a callback function that manipulates DOM in some defined manner.